java/README.md

```diff
@@ -1,14 +1,21 @@
 # XenServerJava usage examples
 
 This folder contains of a number of test programs that can be used as pedagogical
-examples accompanying XenServerJava (com.citrix.hypervisor.xen-api). They are
+examples accompanying XenServerJava (com.xenserver.xen-api). They are
 structured as a Maven project.
 
 ## Dependencies
 
 This code depends on XenServerJava, which in turns depends upon Apache XML-RPC
 by the Apache Software Foundation, licensed under the Apache Software License 2.0.
 
+
+If you have the jar, you can install it using the Maven CLI:
+
+```bash
+mvn install:install-file -Dfile=".\xen-api-XX.YY.ZZ.jar" -DgroupId="com.xenserver" -DartifactId=xen-api -Dversion="XX.YY.ZZ" -Dpackaging=jar -DgeneratePom=true

```

Comment:
Because there is no published 23.12.0 on maven yet, we may want to tell people to go to xen-api repo, generate the SDK and then compile xen-api-XX.YY.ZZ.jar

Reply:
Can we automate publishing to Maven using a github action?

Reply:
Maybe in the future but we can't sign deliverables on GitHub so it'd probably be done internally